impl/src/planning.rs

```diff
@@ -624,12 +653,22 @@ impl<'planner> CrateSubplanner<'planner> {
           )
         }
       }
+
+      // Check if the dependency has been renamed
+      if dep.rename.is_some() {
```

Comment:
Might be more canonical to use:

```
dep.rename.map(|rename| {
  renamed_dep_names.insert(
    dep.name.clone(),
    format!("{}", rename),
  );
})
```

As this would avoid the unwrap

Reply:
Thank you for this! Looks like I had to add `.as_ref()` before `.map` to appease rustc.